testing/btest/Baseline.zam/language.count-underflow/.stderr

```diff
@@ -0,0 +1,5 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
```

Comment:
Mostly curious why the output is not ordered in execution/line order? Is there some sorting filter for the `.zam` baselines?

https://github.com/zeek/zeek/blob/ded011ef02a8876fa7b97d1f84b44954f776bddb/testing/btest/Baseline/language.count-underflow/.stderr#L1-L5

Reply:
I wondered whether anyone would wonder about that - nice eye!  The reason is because ZAM does some constant folding, causing some of the problems to be spotted at compile-time rather than run time.